include/clang/Sema/DeclSpec.h

```diff
@@ -623,6 +631,19 @@ class DeclSpec {
   bool isItypeforanySpecified() const { return FS_itypeforany_specified; }
   SourceLocation getForanySpecLoc() const { return FS_foranyLoc; }
 
+  bool isUnpackSpecified() const { return Unpack_specified; }
+  SourceLocation getUnpackSpecLoc() const { return UnpackLoc; }
+
+  // TODO: add comment about how this method is also used for generics (the comment for existentials is below).
```

Comment:
Yes, these comments need to be cleaned up.

Reply:
Done